src/assets/data/events/all.json

```diff
@@ -1,5 +1,16 @@
 [
     {
+        "title": "Product Design & Ideation",
+        "time": "1/9/2023 8:00 PM",
+        "duration": "120",
+        "type": "Workshop",
+        "desc": "Interested in learning more about product design? Join our beginner-friendly workshop as we discuss UI/UX design, ideation and evaluation, and the product development process. We'll also be taking a closer look at the design tool Figma and going through a quick idea generation/design thinking exercise!",
+        "place": "online",
```

Comment:
Nitpick: noticed in other places the location was capitalized. Any particular reason for not doing so here?

Reply:
Good catch. Fixed!